package-lock.json

```diff
@@ -1,8 +1,7189 @@
 {
   "name": "sbp",
   "version": "3.4.5",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
just noting this is changing the npm lock file version. It should be backwards compatible c.f. https://github.com/swift-nav/libsbp/issues/927

Reply:
Matt has reported that this caused problems with npm 6.14.0.